### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by ElectronicsArchiver** [luni64/EncoderTool#19] [General]
Since the docs are now in the repo, the wiki can be deleted / disabled~
Also README needs some formatting fixes~

**Comment by luni64** [comment]
So, wiki is gone and I fixed the quick - link formatting. Did you see anything else?

**Reply by ElectronicsArchiver** [reply]
Thanks.

> Did you see anything else?

No fixes, just general formatting that could be done.
Will do that in the future I guess~

**Reply by ElectronicsArchiver** [reply]
As a side note, I'd suggest diabling `Projects` & `Actions` since you don't use them.

**Comment by luni64** [comment]
>Will do that in the future I guess~

Feel free to do this whenever you like. I'll need to go over the content the next days. Due to the new platform independence a few changes and additions need to be done. I'll try to stick to your formatting though :-)

**Comment by m-r-m-s** [comment]
@ElectronicsArchiver @luni64 I wanted to say great work on the repo overhaul. It really makes a big difference to the accessibility of information !

**Reply by ElectronicsArchiver** [reply]
Thank you very much~

**Reply by luni64** [reply]
Thanks @m-r-m-s. Credit definitely goes to @ElectronicsArchiver
